ktor-shared/ktor-serialization/ktor-serialization-kotlinx/ktor-serialization-kotlinx-xml/common/src/io/ktor/serialization/kotlinx/xml/XmlSupport.kt

```diff
@@ -21,13 +21,16 @@ import nl.adaptivity.xmlutil.serialization.*
  *
  * See [XML] for more details.
  */
-@OptIn(XmlUtilInternal::class)
 public val DefaultXml: XML = XML {
     repairNamespaces = true
+    // TODO: Remove Suppress and OptIn after update to xmlutil 0.90.2+ or Kotlin 2.0.20+
+    //  https://github.com/pdvrieze/xmlutil/issues/219
+    //  https://youtrack.jetbrains.com/issue/KT-69182/K2-OptIn-on-enum-companion-blocks-enum-constants
+    @Suppress("UnnecessaryOptInAnnotation")
+    @OptIn(XmlUtilInternal::class)
     xmlDeclMode = XmlDeclMode.None
     indentString = ""
     autoPolymorphic = false
-    this.xmlDeclMode
```

Comment:
Wasn't able to track down the reason why this line is here. If there is the reason, please let me know.

Reply:
From what I see it's a redundant line, and it does nothing